README.md

```diff
@@ -5,757 +5,875 @@
 
-You can create an observable from any microstate using the `Observable.from` method.
-The resulting Observable will stream the next microstate for every transition.
-When you subscribe to the observable microstate, you'll immediately receive a microstate through the stream.
-This microstate will have transitions that you can call to cause the next microstate to come through the stream.
+## Transition methods
 
-```js
-import { create } from 'microstates';
-import { Observable } from 'rxjs';
+`transitionTo` is often used by state machine libraries to trigger state transition. Here is an example with xstate library,
 
-let ms = create(Number, 42);
-let observable = Observable.from(ms);
-
-let last;
-
-observable.subscribe(microstate => {
-  last = microstate;
-  console.log(microstate.state);
-});
-// => 42
+```js
+const nextState = lightMachine.transition('green', 'TIMER').value;
 
-last.increment().increment().increment();
-// => 43
-// => 44
-// => 45
+//> 'yellow'
 ```
 
-## Built-in types
+Microstates does not have such a method. Instead, it relies on vanilla JavaScript property lookup. The method invocation is equivalent to calling `transitionTo` with name of the transition. 
 
-Microstates package provides base building blocks for your composing your own types. `Number`, `String`, `Boolean`,
-`Object` and `Array` come with predefined transitions.
+```js
+import { create } from 'microstates';
 
-### `Boolean`
+let lightMachine = create(LightMachine);
 
-`Boolean` type presents a `true` or `false` value. `Boolean` has `toggle` and `set` transitions.
+const nextState = lightMachine.timer();
 
-#### set(value: any) => microstate
+nextState.color.state
+//> 'yellow'
+``` 
 
-Returns a new microstate with boolean value replaced. Value will be coerced with
-`Boolean(value).valueOf()`.
+## Immutable Object vs Immutable Data Structure
 
-```js
-Microstate.create(Boolean).set(true).state;
-// => true
-```
+When you create a state machine with xstate, you create an immutable object. When you invoke a transition on an `xstate` state machine, the value of the object is the ID of the next state. All of the concerns of immutable value change as a result of state change are left for you to handle manually.
 
-#### toggle() => microstate
+Microstates treats value as part of the state machine. It allows you to colocate your state transitions with reducers that change the value of the state.
 
-Returns a new microstate with state of boolean value switched to the opposite.
+# Framework Integrations
 
-```js
-Microstate.create(Boolean).state;
-// => false
+* [React.js](https://github.com/microstates/react)
+* [Ember.js](https://github.com/cowboyd/ember-microstates)
+* Create a PR if you created an integration that you'd like to add to this list.
+* Create an issue if you'd like help integrating Microstates with a framework
 
-Microstate.create(Boolean).toggle().state;
-// => true;
+# `microstates` npm package
```

Comment:
I'm wondering if we should link to this part higher up in the README (like a link to "quick start"?)